torch/_subclasses/fake_tensor.py

```diff
@@ -856,25 +856,6 @@ def __torch_dispatch__(self, func, types, args=(), kwargs=None):
             with self:
                 return decomposition_table[func](*args, **kwargs)
 
-        # prims already wrap FakeTensor inputs to FakeTensor outputs
-        # and do device logic, we dont need do anything but run them
-        # and ensure that Meta kernels are dispatched to (see)
-        # Fake Tensor Dispatch Keys
-        # TODO - we should be use the prim aten impl
-        if (
-            "prims::" in func._schema.name
-            and len(flat_arg_fake_tensors) != 0
-            and hasattr(func, "prim_meta_impl")
-        ):
-            with self:
-                return func.prim_meta_impl(*args, **kwargs)
-
-        if has_symbolic_sizes:
-            if not self.cpp_meta_supports_symint(func):
-                raise RuntimeError(
-                    f"{func} - couldn't find symbolic meta function/decomposition"
-                )
```

Comment:
You probably did not want to remove this second conditional

Reply:
I did want. `symbolic` mode fails with this error because `self.cpp_meta_supports_symint(prim)` is always False.

Reply:
Assuming this is the approach, narrowing to prims seems better